client/cypress/integration/pages/flightsall.spec.js

```diff
@@ -34,8 +34,8 @@ describe("check flights all page", () => {
     cy.get("#select-season").select((year - 1).toString());
 
     // Wait till table was updated
-    cy.url().should("include", `${year - 1}/fluege`);
     cy.get("[data-cy=filter-icon]").should("be.visible");
+    cy.url().should("include", `${year - 1}/fluege`);
```

Comment:
Ich vermute das Problem in den "mehreren" Requests. Die laden ja auch die Seite jeweils neu. 

Evtl. müssen wir da mal den Ablauf in useData auf den Kopf stellen

Reply:
ja, das macht sinn. eigentlich sollte auch der normale spinner kommen wenn eine Seite lädt. den verschleiert das useData auch.